README.md

```diff
@@ -72,117 +72,91 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
 Our [prioritized roadmap][roadmap] of features and utilities that the core team is focusing on.
 
-### 0.60 - June 2022 Update
+### 0.61 - July 2022 Update
 
-In this release, we focused heavily on stability and improvements. Below are some of the highlights!
+This is a lighter release, with a shorter development cycle and focused on stability and improvements.
 
 **Highlights**
 
-- PowerRename and Image Resizer now appear on the new Windows 11 context menu.
-- There's a new PowerToys Run plugin for OneNote. Thanks [@palenshus](https://github.com/palenshus)!
-- FancyZones uses a new zone identification with monitor Id to increase stability and avoid zone resets.
-- AlwaysOnTop now uses rounded corners for highlighting rounded windows on Windows 11.
-- Added settings to PowerToys Run to better control the query results order. Thanks [@jefflord](https://github.com/jefflord)!
+- Quality of life improvements for Always on Top, FancyZones and PowerToys Run.
 
 ### Known issues
 - After installing PowerToys, [the new Windows 11 context menu entries for PowerRename and Image Resizer might not appear before a system restart](https://github.com/microsoft/PowerToys/issues/19124). On some Windows 11 dev channel insider builds, the new context menu entries are not registering correctly and the classic context menu entries will be shown instead.
 - There are reports of users who are [unable to open the Settings window](https://github.com/microsoft/PowerToys/issues/18015). This is being caused by incompatibilities with some applications (RTSS RivaTuner Statistics Server and MSI AfterBurner are known examples of this). If you're affected by this, please check the  linked issue to verify if any of the presented solutions works for you.
 
 ### General
-- Upgraded the Windows App SDK runtimes to 1.1.1. (This was a hotfix for 0.59)
+- Upgraded the Windows App SDK runtimes to 1.1.2.
+- The new Windows 11 context menu entries are now correctly added to Windows 11 dev channel insider builds. (This was a hotfix for 0.60)
+- The old context menu entries are shown alongside the new Windows 11 context menu entries to be compatible with software that overrides the Windows 11 context menu behavior. (This was a hotfix for 0.60)
+- Consolidated C# language version across the solution. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Removed deprecated Segoe icon glyph codes and replaced them with the correct ones.  Thanks [@niels9001](https://github.com/niels9001)!
```

Comment:
```suggestion
- Removed deprecated Segoe icon glyph codes and replaced them with the correct ones.  Thanks [@niels9001](https://github.com/niels9001) and @Jay-o-Way!
```
Plus, I have an additional pr in the waiting room...

Reply:
Sorry about the omission. Adding it.
Regarding the PR, it was marked as ready after we finished testing the release candidate, so it'll likely be included only on next build or if we release a hotfix. Sorry for the inconvenience.

Reply:
Ahhw, I missed the "bell" for the deadline then 🔔